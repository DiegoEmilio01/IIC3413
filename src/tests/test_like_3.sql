SELECT *
FROM T1 as x
WHERE x.a LIKE "test-record-_"