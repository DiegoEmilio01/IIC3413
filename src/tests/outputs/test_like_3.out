x.a,x.b
"test-record-0",30
"test-record-1",31
"test-record-2",32
"test-record-3",33
"test-record-4",34
"test-record-5",35
"test-record-6",36
"test-record-7",37
"test-record-8",38
"test-record-9",39

 got 10 results.
